ingest/bq_ingest_arrays_manifest.sh

```diff
@@ -16,7 +16,7 @@ TMP_PROC="/tmp/manifest_ingest_processed.csv"
 TMP="/tmp/manifest_ingest.csv"
 
 sed 's/,X,/,23X,/g; s/,Y,/,24Y,/g; s/,MT,/,25MT,/g' $MANIFEST_FILE > $TMP_SUB
-sort -t , -k 23n,23 -k24n,24 $TMP_SUB > $TMP_SORTED
+sort -t , -k23n,23 -k24n,24 -k2n,2 $TMP_SUB > $TMP_SORTED
```

Comment:
can you add a few comments in here?  what are these columns, etc.  is column 2 a numeric?